tests/gtest/regularsurface_test.cpp

```diff
@@ -111,7 +111,7 @@ TEST(RegularSurfaceSubscriptTest, DoubleRowOutOfRange) {
 
     EXPECT_THAT([&]() { surface[above_upper_row]; },
                 testing::ThrowsMessage<std::runtime_error>(
-                    testing::HasSubstr("Row out of range")));
+                    testing::HasSubstr("operator[]: index out of range")));
```

Comment:
Are you sure you wanted to change stuff here to index?

To me "index" was an example as for the first one, main part was `operator[]:` . I am fine if here we have row/column still where it was row/column before.
But if you want index - that's fine